### PR TITLE
#114 Update dependencies, refactor router and small table refactor

### PR DESCRIPTION
**Pull request by adrigardi90**
Dependencies updated with the new react-router-dom version and the appropriate refactor (react-router-dom re-exports all of react-router's exports). Furthermore an small refactor on the table.

The readme needs to be updated once the changes have been approved.

@brauliodiez

### CODE REVIEW

File: 05 Presentational Comp/src/router.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+import { Route, BrowserRouter } from 'react-router-dom';
 import { App } from './app';
 import { About, MembersPage } from './components';
 
 export const AppRouter: React.StatelessComponent<{}> = () => {
   return (
-    <Router history={hashHistory}>
-      <Route path="/" component={App} >
-        <IndexRoute component={About} />
+    <BrowserRouter>
```
Comment (arp82) [inline 05 Presentational Comp/src/router.tsx]: Since the old version used hashHistory, should it not be HashRouter instead of BrowserRouter?

Comment (adrigardi90) [reply 05 Presentational Comp/src/router.tsx]: Absolutely right. I'll change it

**Comment by adrigardi90** [comment]
HashRouter added instead of BrowserRouter

File: 05 Presentational Comp/src/components/members/memberBody.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { MemberEntity } from '../../model';
+import { MemberRow } from './memberRow';
+
+interface Props {
+    members: MemberEntity[];
+}
+
+export const MemberBody: React.StatelessComponent<Props> = ({ members }) => {
+
```
Comment (Nasdan) [inline 05 Presentational Comp/src/components/members/memberBody.tsx]: Remove extra space

File: 05 Presentational Comp/src/components/members/memberHeader.tsx [hunk 3]
```diff
@@ -2,10 +2,13 @@ import * as React from 'react';
...
-      <th>Name</th>
-    </tr>
+    <thead>
+      <tr>
+        <th>Avatar</th>
+        <th>Id</th>
+        <th>Name</th>
+      </tr>
+    </thead>
+
```
Comment (Nasdan) [inline 05 Presentational Comp/src/components/members/memberHeader.tsx]: Remove extra space

**Comment by Nasdan** [comment]
Good job, only it's pending to update Readme

**Comment by adrigardi90** [comment]
Shall I update the Readme? Or are you going to do it?

**Comment by Nasdan** [comment]
Could you please update it?

**Comment by adrigardi90** [comment]
Readme updated

**Pull request closed without merging**